webdriver/tests/bidi/browsing_context/locate_nodes/locator.py

```diff
@@ -85,55 +85,55 @@ async def test_find_by_locator(bidi_session, inline, top_context, type, value):
          "matchType": "full",
          "maxDepth": 0,
          "value": "foobarbarbaz"
-     }, ["body"]),
+     }, ["html"]),
```

Comment:
I wonder if that is correct since it would mean that the start node is also a result node although it does seem to be what the spec expects for inner text